test/skimNtuple2018.cpp

```diff
@@ -2492,11 +2499,64 @@ int main (int argc, char** argv)
           Long64_t trgNotOverlapFlag = (Long64_t) theBigTree.mothers_trgSeparateMatch->at(chosenTauPair);
           bool passTrg = trigReader.checkOR (pairType,triggerbit, &pass_triggerbit, matchFlag1, matchFlag2, trgNotOverlapFlag, goodTriggerType1, goodTriggerType2, tlv_firstLepton.Pt(), tlv_firstLepton.Eta(), tlv_secondLepton.Pt(), tlv_secondLepton.Eta()) ;
 
+     	  if (!isMC && passTrg) {
+	    if(theBigTree.RunNumber < 317509){
+	      if (DEBUG) 
+		{ 
+		  cout << "@@@ Run in period without HPS triggers "<<endl;
+		  cout << "isHPSTauTrigger = " << std::bitset<16>(isHPSTauTrigger) << endl;
+		  cout << "pass_triggerbit = " << std::bitset<16>(pass_triggerbit) << endl;
+		}
+	      if ((pass_triggerbit & (~ isHPSTauTrigger)) == 0) {
+		if (DEBUG) cout << "only HPS trigger fired, reject"<<endl;
+		passTrg = false; 
+	      }
+	    }else{
+	      if (DEBUG) 
+		{ 
+		  cout << "@@@ Run in period with HPS triggers "<<endl;
+		  cout << "isHPSTauTrigger = " << std::bitset<16>(isHPSTauTrigger) << endl;
+		  cout << "isTauTrigger =    " << std::bitset<16>(isTauTrigger) << endl;
+		  cout << "pass_triggerbit = " << std::bitset<16>(pass_triggerbit) << endl;
+		}
+	      
+	      if ((pass_triggerbit & (~ isTauTrigger)) == 0) {
+		if (DEBUG) cout << "only HPS trigger fired, reject"<<endl;
```

Comment:
the debug comment should be "only ***non-HPS*** trigger fired, reject"

Reply:
done